test/verifying_paymaster.test.ts

```diff
@@ -61,7 +71,7 @@ describe('EntryPoint with VerifyingPaymaster', function () {
         const sig = await offchainSigner.signMessage(arrayify('0xdead'))
         wrongSigUserOp = await fillAndSign({
           sender: account.address,
-          paymasterAndData: hexConcat([paymaster.address, sig])
+          paymasterAndData: hexConcat([paymaster.address, '0x000000007B9F2E9D', '0x0000000000001234', sig])
```

Comment:
please make it a "recognizable" value, or a constant. I read understand "0x1234" as "just a dummy value we can put in and later verify", but "7B9F2E9D" is unclear

Reply:
Good point i'll use more dummy-like values and use constant variables to make things clear